test.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+import logging
+import os
+import random
+import sys
+import numpy as np
+import torch
+import torch.backends.cudnn as cudnn
+import torch.nn as nn
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from datasets.crack_datasets import Crack_Datasets
+from networks.LECSFormer import LECSFormer
+from config import get_config
+from collections import OrderedDict
+import cv2
```

Comment:
Please organize the import according to 
first group: default python package (sys, os, etc)
second group: installed package (numpy, pandas, torch)
third group: package in this repo

In each group, the package should be in alphabetical order.
You may ask chat gpt to reorganize this for you.